experiment/measurer/coverage_over_time_utils.py

```diff
@@ -0,0 +1,189 @@
+# Copyright 2020 Google LLC
```

Comment:
Err...I appreciate the rename but the two suggestions I made were alternatives. So please just rename this module detailed_coverage_data_utils.py. Thanks!

Reply:
Oops, sorry for that :)

Reply:
Done. Thanks!